fitsio/fitsio_pywrap.c

```diff
@@ -3603,17 +3626,13 @@ PyFITSObject_read_columns_as_rec(struct PyFITSObject* self, PyObject* args) {
 
     int status=0;
 
-    PyObject* columnsobj=NULL;
-    PyObject* array=NULL;
-
-    PyObject* rowsObj=NULL;
-    PyObject* sortindObj=NULL;
+    PyObject *columns_obj, *array_obj, *rows_obj, *sortind_obj;
```

Comment:
Did we want to get rid of the NULL initialization here?

Reply:
They are immediately read into by the module initialization, so its OK in this case. I did it for readability. But if you don't like I'm happy to set NULL

Reply:
by module init I meant things like `PyArg_ParseTuple`

Reply:
Never mind, I'll just set NULL